build.gradle

```diff
@@ -18,6 +20,7 @@ dependencies {
     implementation 'commons-cli:commons-cli:1.4'
     implementation 'io.github.classgraph:classgraph:4.8.46'
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile "org.jetbrains.kotlinx:kotlinx-serialization-runtime:0.13.0"
```

Comment:
Should be `implementation`, not `compile` (as should the line above; I'll fix that).